src/matching.py

```diff
@@ -349,6 +349,15 @@ class RuleStrategy:
     and they return None when there is no match
     """
 
+    @staticmethod
+    def match_verbatim(verbatims: list, verbatim_id: str, **kwargs):
+        """
+        Match exactly to a verbatim message's ID.
+        """
+        for verbatim in verbatims:
+            if verbatim["id"] == verbatim_id:
+                return {"operation": "verbatim_response", "verbatim": verbatim}
```

Comment:
Btw, I wrestled a bit with whether I like having `"operation": "verbatim_response"` represent multiple things, since operations were all very distinct before, but after sitting with it a bit, I really think it's the right choice!

👍 👍

Reply:
Also, what do you think about just making it `"operation": "verbatim"` I feel like "response" is a little redundant, since everything has a response by nature of the bot.

For ex, `all_the_plans` vs `all_the_plans_response`. 

Curious for any thoughts here though if there's a specific reason you leaned toward `verbatim_response` :)

Reply:
> Also, what do you think about just making it "operation": "verbatim" I feel like "response" is a little redundant, since everything has a response by nature of the bot.

Sounds fine to me, I'll make the change.